src/scripts.js

```diff
@@ -1,17 +1,83 @@
 // This is the JavaScript entry file - your code begins here
 // Do not delete or rename this file ********
-
-console.log(userData,"<>>>>userData")
+console.log(userData, "<>>>>userData");
 // An example of how you tell webpack to use a CSS file
-import './css/styles.css';
+import "./css/styles.css";
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+import "./images/turing-logo.png";
 
-console.log('This is the JavaScript entry file - your code begins here.');
+console.log("This is the JavaScript entry file - your code begins here.");
 
 // An example of how you tell webpack to use a JS file
 
-import userData from './data/users';
+import userData from "./data/users";
+
+import UserRepository from "./UserRepository";
+//variables
+const userRepository = new UserRepository(userData);
+const randomIndex = getRandomIndex();
+const userInfo = userRepository.findUserData(randomIndex);
+const user = new User(userInfo);
+
+// Query Selectors by class Name
+const welcomeMessage = document.getElementsByClassName("welcome-message");
```

Comment:
Update all of these to document.querySelector, this is selecting for multiple elements